processors/base/helpers.py

```diff
@@ -58,6 +58,16 @@ def default(self, obj):
         return json.JSONEncoder.default(self, obj)
 
 
+def clean_list(raw_list):
+    """Remove falsy values from list.
+    """
+    cleaned_list = []
+    for value in raw_list:
+        if value:
+            cleaned_list.append(value)
+    return cleaned_list
```

Comment:
Yea here it could be used without code complication. Just because of my rule to do not use non obvious for all or control-flow complicating language concepts sometimes I miss opportunities like this.